OpenRA.Game/Actor.cs

```diff
@@ -202,6 +222,11 @@ public IEnumerable<T> TraitsImplementing<T>()
 			return World.TraitDict.WithInterface<T>(this);
 		}
 
+		public T TraitInfo<T>()
```

Comment:
Ok, I will put it back, but we'll need to think about changing the constructor to supply those dependencies in a future PR.